config/urls.py

```diff
@@ -41,6 +42,12 @@
     path("pipelines/", include("hexa.pipelines.urls", namespace="pipelines")),
     path("metrics/", include("hexa.metrics.urls", namespace="metrics")),
     path("comments/", include("hexa.comments.urls")),
+    # Order matters, we override the default logout view defined later
```

Comment:
We should just add 1-2 sentences of explanation here : why do we do this? And Add a `  #TODO: use API when available https://github.com/jupyterhub/jupyterhub/issues/3688`)